### ISSUE TITLE
Merge `merge#249` into `master`

### ISSUE THREAD

**Issue by wiztigers** [TypeCobolTeam/TypeCobol#303]
**CHILD OF #249**

Current `master` should become a legacy/museum branch that we'll keep "just in case".
Current `merge#249` must become our new `master`, and then be deleted to avoid confusion.

**Comment by wiztigers** [comment]
From ae5f25e007241bb1809c3a50a95705825bb7c5cc on, formerly called `branch#249` is our new `master`.

I first created a new branch `legacy-20161004` which is exactly like `master` was.
Do NOT commit any features on this branch ! It is just there as a remembering of what was.
I created it just in case there is something hugely problematic with #249 that I introduced while merging, so we can revert back. 

Now, about the merge in particular.
The goal was to keep `master` as an ancestor : I couldn't simply overwrite one branch with the other, for this would have messed with the history of everyone who had cloned our public repository.
Here is the merge strategy I employed :
```
git checkout merge#249
git merge --strategy=ours --no-commit master  # keep the content of the branch
git commit                                    # record a merge
git checkout master
git merge merge#249                           # fast-forward master up to the merge
```

Now, to avoid confusion (for example regarding currently ongoing pull request #299), we have to delete the branch `merge#249`. However, it is marked "protected" by Github.
@smedilol, could you unprotect it, so we can delete it.
Regarding branch deletion, it seems important to delete it [through the PR](https://help.github.com/articles/deleting-unused-branches/), so we keep the option to restore it later.

**Comment by smedilol** [comment]
Done, you can delete the branch

**Comment by wiztigers** [comment]
Pull request's page didn't propose me to delete any branch.
So, I deleted `merge#249` through the branches pages.
Github indeed offered me to restore the branch after I deleted it ... until I reloaded the page. Now, it seems we cannot restore this branch anymore.
I'd say ... whatever. :)

As I side note, deleting branch `merge#249` automatically closes pull request #299, which was based on it. PR #299 cannot be reopened anymore. Now we know.
